modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/layout/display/page/CProductLayoutDisplayPageProvider.java

```diff
@@ -72,10 +78,26 @@ public String getClassName() {
 		getLayoutDisplayPageObjectProvider(long groupId, String urlTitle) {
 
 		try {
+			long classNameId = _portal.getClassNameId(CProduct.class);
+
+			Group group = _groupLocalService.getGroup(groupId);
+
+			Group companyGroup = _groupLocalService.getCompanyGroup(
+				group.getCompanyId());
+
+			FriendlyURLEntry friendlyURLEntry =
+				_friendlyURLEntryLocalService.fetchFriendlyURLEntry(
+					companyGroup.getGroupId(), classNameId, urlTitle);
```

Comment:
We can remove the variable.
```suggestion
			Group group = _groupLocalService.getGroup(groupId);

			Group companyGroup = _groupLocalService.getCompanyGroup(
				group.getCompanyId());

			FriendlyURLEntry friendlyURLEntry =
				_friendlyURLEntryLocalService.fetchFriendlyURLEntry(
					companyGroup.getGroupId(),
					_portal.getClassNameId(CProduct.class), urlTitle);
```